cmd/root.go

```diff
@@ -285,6 +278,38 @@ func parseConfig(cmd *cobra.Command, conf *proxy.Config, args []string) error {
 	return nil
 }
 
+// parseBoolOpt parses a boolean option from the query string
+//  name the name of the option in the query string
```

Comment:
This and everything below is available from the type signature and therefore unnecessary, with the exception of t/true and f/false.

Reply:
done